README.md

```diff
@@ -31,7 +31,7 @@ Active development occurs in this repository, which means some contracts in it m
 
 ## Pre-requisites
 
-The build & test instructions below should work out of the box with Node ^14.18.0. (Please note that it needs Node 14 specifically, and will NOT work with Node 16 or higher. Minor version should be at least 18).
+The build & test instructions below should work out of the box with Node 18. More specifically, it is recommended to use the LTS version ^18.15; Node 19 is not supported.
```

Comment:
```suggestion
The build & test instructions below should work out of the box with Node 18. More specifically, it is recommended to use the LTS version 18.15; Node 19 and higher are not supported.
```
We need 18.15 exactly, right? And you mentioned 20's out :) TMI to say why we need 15 specifically?

Reply:
How about now?